backend/src/configuration/models.rs

```diff
@@ -15,9 +15,7 @@ impl From<u16> for FlukePort {
 }
 
 pub struct FlukeConfiguration {
-    /// REQUIRED! There is not default value
     pub database_url: String,
-    /// Default 8000
```

Comment:
why did you remove these

Reply:
Described what the code was doing when the code itself did a good job of that